subsys/bluetooth/host/Kconfig

```diff
@@ -109,16 +109,29 @@ config BT_DRIVER_RX_HIGH_PRIO
 if BT_HCI_HOST
 
 config BT_HOST_CRYPTO
-	# Hidden option that compiles in random number generation and AES
-	# encryption support using TinyCrypt library if this is not provided
-	# by the controller implementation.
+	# Hidden option that compiles in AES encryption support using TinyCrypt
+	# library if this is not provided by the controller implementation.
 	bool
 	default y if !BT_CTLR_CRYPTO
 	select TINYCRYPT
 	select TINYCRYPT_AES
+
+config BT_HOST_CRYPTO_RANDOM
```

Comment:
Can we call this `BT_HOST_CRYPTO_PRNG`, since that's really what it's affecting: when enabled we use a PRNG to generate random data, and when disabled we rely on HCI_LE_Rand.